README.md

```diff
@@ -207,14 +207,16 @@ $ omf install https://github.com/jhillyerd/plugin-git
 | g            | `git`                                                       |
 | gap          | `git apply`                                                 |
 | gbl          | `git blame -b -w`                                           |
-| gcf          | `git config --list`                                         |
+| gcf          | `git config --list`                                          |
 | gcl          | `git clone`                                                 |
-| gclean       | pristine working directory: reset and force clean           |
+| gclean       | `git clean -di`                                             |
+| gclean!      | `git clean -dfx`                                            |
+| gclean!!     | `git reset --hard; and git clean -dfx`                      |
 | gcp          | `git cherry-pick`                                           |
 | gcpa         | `git cherry-pick --abort`                                   |
 | gcpc         | `git cherry-pick --continue`                                |
 | gignore      | `git update-index --assume-unchanged`                       |
-| gignored     | list temporarily ignored files                              |
+| gignored     | list temporarily ignored files                               |
```

Comment:
```suggestion
| gignored     | list temporarily ignored files                              |
```

Reply:
It's strange. Previously I change them because it looks alignment in my vscode. 
I check it again in my vim after you raise this problem. It turns out there is extra space. It displays differently in my vim and vscode. 
Anyway, I have fixed them.